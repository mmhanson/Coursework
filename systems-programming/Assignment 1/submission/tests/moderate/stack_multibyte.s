main:
	subl	$4, %esp
	movl	$-21555, %ebx
	movl	%ebx, 0(%esp)
	movl	0(%esp), %ecx
	printr	%ecx
	pushl	%ebx
	popl	%edx
	printr	%edx
