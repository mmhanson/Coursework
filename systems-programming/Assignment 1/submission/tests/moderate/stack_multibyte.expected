-21555 (0xffffabcd)
-21555 (0xffffabcd)
